charts/core/src/ChartContext/ChartContext.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { createContext, PropsWithChildren, useContext } from 'react';
+
+import { ChartContextType, ChartProviderProps } from './ChartContext.types';
+
+export const ChartContext = createContext<ChartContextType | undefined>(
+  undefined,
+);
+
+export const ChartProvider = ({
+  children,
+  chartOptions,
+  updateChartOptions,
+  addChartSeries,
+}: PropsWithChildren<ChartProviderProps>) => {
```

Comment:
The type file includes `removeChartSeries` but we don't use that here

Reply:
Good catch! This isn't really doing anything yet but might as well pass it down.

I've also updated types to just use `ChartContextType` since really the provider and context should have the same types.